src/_guides/language/evolution.md

```diff
@@ -286,11 +286,10 @@ Because migrating your apps and packages
 to null safety might take a while,
 Dart uses language versioning to support
 using non-null-safe code alongside null-safe code.
-
-{% comment %}
-  Once /null-safety/migrating-to-null-safety exists,
-  the previous paragraph should link to it.
-{% endcomment %}
+For an example of how an app or package can migrate to
+a new language version
+with an incompatible feature—like was introduced with null safety—see 
```

Comment:
```suggestion
with an incompatible feature (null safety, for example), see 
```

I had to read this sentence multiple times because I thought it said "feature-like". It didn't *look* like an mdash until I went to edit the text.